src/shared/DotnetTool/pack-tool.sh

```diff
@@ -53,36 +52,25 @@ GITLAB_UI_SRC="$SRC/shared/GitLab.UI.Avalonia"
 DOTNET_TOOL="shared/DotnetTool"
 PROJ_OUT="$OUT/$DOTNET_TOOL"
 
-PACKAGE="$ROOT/nuget"
-CONFIGURATION="${CONFIGURATION:=Release}"
+CONFIGURATION="${CONFIGURATION:=Debug}"
 
 # Build parameters
 FRAMEWORK=net6.0
 
 # Outputs
-PAYLOAD="$PROJ_OUT/payload/$CONFIGURATION"
-SYMBOLOUT="$PROJ_OUT/payload.sym/$CONFIGURATION"
-
-# Cleanup payload directory
-if [ -d "$PAYLOAD" ]; then
-    echo "Cleaning existing payload directory '$PAYLOAD'..."
-    rm -rf "$PAYLOAD"
-fi
-
-# Cleanup symbol directory
-if [ -d "$SYMBOLOUT" ]; then
-    echo "Cleaning existing symbols directory '$SYMBOLOUT'..."
-    rm -rf "$SYMBOLOUT"
-fi
-
-# Cleanup package directory
-if [ -d "$PACKAGE" ]; then
-    echo "Cleaning existing package directory '$PACKAGE'..."
-    rm -rf "$PACKAGE"
+OUTDIR="$PROJ_OUT/nupkg/$CONFIGURATION"
+IMGOUT="$OUTDIR/images"
+PAYLOAD="$OUTDIR/payload"
+SYMBOLOUT="$OUTDIR/payload.sym"
```

Comment:
Everything sits under `$OUTDIR` now which is `/out/shared/DotnetTool/nupkg/$CONFIGURATION`.